common/s3util.cpp

```diff
@@ -254,6 +310,10 @@ void S3Util::listObjectsHelper(const string& prefix, const string& delimiter,
     }
   } else {
     if (error_message != nullptr) {
+      Stats::get()->Incr(folly::sformat("s3_list_objects_helper_error response_code={} exception_name={} should_retry={}", 
+        static_cast<int32_t>(listObjectResult.GetError().GetResponseCode()),
+        listObjectResult.GetError().GetExceptionName(), 
+        listObjectResult.GetError().ShouldRetry()));
```

Comment:
Do you have a sample error log after this change?

Reply:
@kangnanli this was in my previous pull request https://github.com/pinterest/rocksplicator/pull/602

Reply:
![Screen Shot 2022-03-25 at 10 55 31 AM](https://user-images.githubusercontent.com/1491679/160667541-8a7cd600-8bfc-47d1-90bb-30d413fd628e.png)